ggml-sycl.cpp

```diff
@@ -9611,7 +6218,8 @@ static void norm_f32_sycl(const float *x, float *dst, const int ncols,
                     });
         });
     } else {
-        const int work_group_size = g_work_group_size;
+        // FIXME: 1024 from cuda
+        const int work_group_size = 1024;
```

Comment:
As opposed to hard coding this could we use a macro? This would make this easier to fix/update in the future. I.e. add to `common.hpp`
```
// FIXME: 1024 from cuda
#define WG_SIZE 1024
```
and in `ggml-sycl.cpp` use
```
const int work_group_size = WG_SIZE;
```

Reply:
fixed in https://github.com/ggerganov/llama.cpp/pull/7710/commits/28eaafc16617079ad439fdd4d5b020797976f028